libs/output-mapping/src/Writer/Helper/SliceCommandBuilder.php

```diff
@@ -22,12 +25,17 @@ public static function createProcess(
             '--table-output-path=' . $outputDir->getPathname(),
             '--table-output-manifest-path=' . $outputDir->getPathname() . '.manifest',
             '--gzip=true',
+            '--input-size-low-exit-code=' . self::SLICER_SKIPPED_EXIT_CODE,
         ];
 
         if ($inputManifestFile) {
             $command[] = '--table-input-manifest-path=' . $inputManifestFile->getPathname();
         }
 
+        if ($inputSizeThreshold) {
```

Comment:
Pro potřeby testů

Reply:
A proč to neudělat povinný pro potřeby všech ?

Reply:
A proc jo?

Reply:
@odinuv Ja jsem to tak mel vlastne puvodne i pripraveny. Ale kdyz by byl povinnej, musel bych ho natlacit pred `$inputManifestFile` a to se mi nelibilo. Proto jsem to udelal radsi takhle. Pokud na tom budes trvat, tak to udelam :-)

Reply:
aha, ok, ne netrvám na tom, byla to jen otázka na to jestli to má nějakej rozmnej důvod